### ISSUE TITLE
Got status code 500 (unoconvert error: exit status 1) when converting from rebuild 3.18 image

### ISSUE THREAD

**Issue by jmjjg** [libreofficedocker/libreoffice-unoserver#5]
Hello,

I was using the `libreofficedocker/libreoffice-unoserver:3.18` image that was built about 5 weeks ago to convert ODT document to PDF document.

However, it seems that you rebuilt the existing docker images about 2 days ago (October 22 2023) and since then, when pulling the new docker image, I get an HTTP status 500 (`unoconvert error: exit status 1`) when trying to convert the document.

Here is the ODT document I'm trying to convert: [template.odt](https://github.com/libreofficedocker/libreoffice-unoserver/files/13113386/template.odt)

```
php-office-clients-unoserver-1  | INFO:unoserver:Starting unoserver.
php-office-clients-unoserver-1  | INFO:unoserver:Command: libreoffice --headless --invisible --nocrashreport --nodefault --nologo --nofirststartwizard --norestore -env:UserInstallation=file:///tmp/tmpunsb3tb4 --accept=socket,host=127.0.0.1,port=2002,tcpNoDelay=1;urp;StarOffice.ComponentContext
php-office-clients-unoserver-1  | INFO:unoserver:Server PID: 108
php-office-clients-unoserver-1  | 
php-office-clients-unoserver-1  |    __  __
php-office-clients-unoserver-1  |   / / / /___  ____  ________  ______   _____  _____
php-office-clients-unoserver-1  |  / / / / __ \/ __ \/ ___/ _ \/ ___/ | / / _ \/ ___/
php-office-clients-unoserver-1  | / /_/ / / / / /_/ (__  )  __/ /   | |/ /  __/ /
php-office-clients-unoserver-1  | \____/_/_/_/\____/____/\___/_/   _|___/\___/_/____
php-office-clients-unoserver-1  |       / __ \/ ____/ ___/_  __/  /   |  / __ \/  _/
php-office-clients-unoserver-1  |      / /_/ / __/  \__ \ / /    / /| | / /_/ // /
php-office-clients-unoserver-1  |     / _, _/ /___ ___/ // /    / ___ |/ ____// /
php-office-clients-unoserver-1  |    /_/ |_/_____//____//_/    /_/  |_/_/   /___/
php-office-clients-unoserver-1  | 
php-office-clients-unoserver-1  | - Alpine Linux version: 3.18.3
php-office-clients-unoserver-1  | - LibreOffice 7.5.5.2 50(Build:2)
php-office-clients-unoserver-1  | - Python version: Python 3.11.5
php-office-clients-unoserver-1  | - openjdk 11.0.20 2023-07-18
php-office-clients-unoserver-1  | - OpenJDK Runtime Environment (build 11.0.20+8-alpine-r0)
php-office-clients-unoserver-1  | - OpenJDK 64-Bit Server VM (build 11.0.20+8-alpine-r0, mixed mode)
php-office-clients-unoserver-1  | 
php-office-clients-unoserver-1  | Starting REST API server...
php-office-clients-unoserver-1  | + wait-for-it --quiet -t 30 localhost:2002 --strict -- unoserver-rest-api --unoconvert-bin /usr/bin/unoconvert-lockrun --unoconvert-timeout 1m
php-office-clients-unoserver-1  | unoserver-rest-api 2023/10/24 09:54:17 Server working directory '/tmp/unodata-2919888351'
php-office-clients-unoserver-1  | unoserver-rest-api 2023/10/24 09:54:17 Server is running at 0.0.0.0:2004
php-office-clients-unoserver-1  | unoserver-rest-api 2023/10/24 09:54:17 Server is running pid is 164
php-office-clients-unoserver-1  | unoserver-rest-api 2023/10/24 09:54:35 Command: /usr/bin/unoconvert-lockrun [--interface=127.0.0.1 --port=2002 /tmp/unodata-2919888351/3714119047-vnd.oasis.opendocument.text /tmp/unodata-2919888351/2065119147-vnd.oasis.opendocument.text.pdf]
php-office-clients-unoserver-1  | unoserver-rest-api 2023/10/24 09:54:35 unoconvert error: exit status 1
php-office-clients-unoserver-1  | [GIN] 2023/10/24 - 09:54:35 | 500 |   63.848963ms |      172.20.0.7 | POST     "/request"
```

**Comment by socheatsok78** [comment]
I've found that the `unoserver` have released a new version v2.0. This image is not supported the v2.0 yet. 

Now I'm pinning the `unoserver==1.6` which is the previous version.

Will update support for v2.0 later.

**Comment by jmjjg** [comment]
That does fix it, thank you very much, especially for the really fast fix!
